README.md

```diff
@@ -14,7 +16,19 @@ The Kafka Sink will create the default partition for a given topic, if the topic
 Kafka cluster.
 The publishing topic and partition can be a dynamic value taken from the Siddhi event."
 
-#### Prerequisites for using the feature
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-kafka">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-kafka/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-kafka/issues">Issue tracker</a>
+
+## Latest API Docs
+
+Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-kafka/api/4.0.3-SNAPSHOT">4.0.3-SNAPSHOT</a>.
+
+## How to use
+
+### Prerequisites for using the feature
  - Download and install Kafka.
```

Comment:
It's not DAS it's SP :p, and have this under how to use in SP section

Reply:
requested change is done in 34e69cc959dc3e1f6c85843569bf68ce4a866bde